packages/hl7v2-server/src/app.ts

```diff
@@ -0,0 +1,47 @@
+import { Hl7Server } from "@medplum/hl7";
+import * as dotenv from "dotenv";
+import { ConsoleLogger, Logger } from "./logger";
+dotenv.config();
+
+const HL7_PORT = process.env.HL7_PORT ? parseInt(process.env.HL7_PORT) : 2575;
+
+async function createHl7Server(logger: Logger): Promise<Hl7Server> {
+  const server = new Hl7Server(connection => {
+    logger.info("Connection received");
+
+    connection.addEventListener("message", ({ message }) => {
+      logger.debug(
+        `## New Message from ${connection.socket.remoteAddress}:${connection.socket.remotePort} ##`
+      );
+      message.segments.forEach(segment => {
+        logger.debug("Segment:", segment.toString());
+      });
+      logger.debug("\n\n");
+      connection.send(message.buildAck());
+    });
+
+    connection.addEventListener("error", error => {
+      if (error instanceof Error) {
+        logger.error("Connection error:", error);
```

Comment:
Sentry? Or just logs for now?

Reply:
Just logs for now, will do Sentry in another PR. This one was getting big enough as it is.